--- xf86-video-apm-1.2.5/src/apm.h~	2012-07-17 04:52:01.000000000 +0000
+++ xf86-video-apm-1.2.5/src/apm.h	2013-03-23 18:39:50.667774586 +0000
@@ -33,9 +33,6 @@
 /* All drivers initialising the SW cursor need this */
 #include "mipointer.h"
 
-/* All drivers implementing backing store need this */
-#include "mibstore.h"
-
 /* I2C support */
 #include "xf86i2c.h"
 
--- xf86-video-apm-1.2.5/src/apm_driver.c~	2012-07-17 06:52:01.000000000 +0200
+++ xf86-video-apm-1.2.5/src/apm_driver.c	2013-04-06 02:19:34.853758056 +0200
@@ -1784,7 +1784,6 @@ ApmScreenInit(SCREEN_INIT_ARGS_DECL)
     }
 #endif
 
-    miInitializeBackingStore(pScreen);
     xf86SetBackingStore(pScreen);
     xf86SetSilkenMouse(pScreen);
 
